server/lib/nl/fulfillment/types.py

```diff
@@ -71,9 +71,9 @@ class ChartVars:
   # Set if is_topic_peer_group is set.
   svpg_id: str = ''
 
-  # Map of sv to facet metadata that has data for this sv. Used by LINE tiles
+  # Map of sv to place to facet metadata that has data for this sv. Used by LINE tiles
   # when there is a date specified, and by the "is_coplottable" check.
-  sv_exist_facet = Dict[str, Dict[str, str]]
+  sv_exist_facet = Dict[str, Dict[str, Dict[str, str]]]
```

Comment:
Should we use a dataclass now?  So its more clear everywhere we pass around?

Reply:
discussed offline to have a followup PR to try to enhance https://github.com/datacommonsorg/website/blob/368d8d7cdfc1cd5086809d15b39c820d1fe763e7/server/lib/nl/fulfillment/types.py#L122-L123 to save facet metadata, so won't make the structure change now